catalyst-gateway/bin/src/event_db/legacy/queries/event/follower.rs

```diff
@@ -0,0 +1,151 @@
+//! Follower Queries
+
+use async_trait::async_trait;
+use cardano_chain_follower::Network;
+use chrono::TimeZone;
+
+use crate::event_db::{Error, EventDB};
+
+/// Block time
+pub type BlockTime = i64;
+/// Slot
+pub type SlotNumber = i64;
+/// Epoch
+pub type EpochNumber = i64;
+/// Block hash
+pub type BlockHash = String;
+/// Unique follower id
+pub type MachineId = String;
+/// Time when a follower last indexed
+pub type LastUpdate = chrono::DateTime<chrono::offset::Utc>;
+
+#[async_trait]
+#[allow(clippy::module_name_repetitions)]
+/// Follower Queries Trait
+pub(crate) trait FollowerQueries: Sync + Send + 'static {
```

Comment:
same

Reply:
i'll do a separate PR for workspace and trait refactoring suggestions.